Sources/KIF/Classes/KIFUIViewTestActor.m

```diff
@@ -162,25 +163,54 @@ - (BOOL)acknowledgeSystemAlert;
 {
     return [self.actor acknowledgeSystemAlert];
 }
+
+- (void)acknowledgeSystemAlertWithCompletionHandler:(void (^)(BOOL, NSError*))completionHandler {
+    @try {
```

Comment:
This will likely lead to a memory leak in the test.

Reply:
I think we should just let it fail if the error comes in like this

Reply:
Will the exception be caught by the XCTTestCase and continue running the next test, or will it crash the app?

Reply:
If we don't catch it, XCTest don't catch it neither. Because it is in a `dispatch_queue` it bubbles up to the `@main` of the project and stop the execution of all tests. Example:
<img width="1064" alt="Screenshot 2023-02-22 at 08 21 48" src="https://user-images.githubusercontent.com/3220409/220461437-cd32d124-0c76-4d4b-a914-bac28698c6b4.png">